package-feeds/all/Packages

```diff
@@ -486,3 +486,71 @@ UserVisible: yes
 Version: 1.0.0.2
 VisibleForRuntimeDeployment: no
 
+Architecture: windows_x64
+Built-Using: nipkg
+CompatibilityVersion: 240500
+Description: Measurement plug-in example to perform a measurement using an DMM.
+DisplayName: DMM Measurement HAL (Py)
+Filename: https://github.com/NI-Measurement-Plug-Ins/abstraction-layer-python/releases/download/1.0.0/dmm-measurement-hal_1.0.0_windows_x64.nipkg
+MD5sum: 4922b3048a1c273cd340f38f21639fbe
+Maintainer: National Instruments
+Package: dmm-measurement-hal
+Plugin: file
+Section: Add-Ons
+Size: 13618
+StoreProduct: no
+UserVisible: yes
+Version: 1.0.0
+VisibleForRuntimeDeployment: no
+
+Architecture: windows_x64
+Built-Using: nipkg
+CompatibilityVersion: 240500
+Description: Measurement plug-in example to source DC voltage using NI SMU and measure the same using an NI SMU or DMM.
+DisplayName: Source Measure DC Voltage FAL (Py)
+Filename: https://github.com/NI-Measurement-Plug-Ins/abstraction-layer-python/releases/download/1.0.0/source-measure-dc-voltage-fal_1.0.0_windows_x64.nipkg
+MD5sum: 70a92aaacf47bf3f163f68e1f11199dd
+Maintainer: National Instruments
+Package: source-measure-dc-voltage-fal
+Plugin: file
+Section: Add-Ons
+Size: 15258
+StoreProduct: no
+UserVisible: yes
+Version: 1.0.0
+VisibleForRuntimeDeployment: no
+
+Architecture: windows_x64
+Built-Using: nipkg
+CompatibilityVersion: 240500
+Description: mycompany-myproduct
+DisplayName: DmmMeasurementHAL
+Filename: https://github.com/NI-Measurement-Plug-Ins/abstraction-layer-labview/releases/download/v1.0.0.1/ni-dmm-measurement-hal_1.0.0.1_windows_x64.nipkg
+MD5sum: 1f365c4a53341906fec231c007b5ba55
+Maintainer: Github Community <santhosh.ramaraj@ni.com>
+Package: ni-dmm-measurement-hal
+Plugin: file
+Section: Add-Ons
+Size: 20029564
+StoreProduct: no
+UserVisible: yes
+Version: 1.0.0.1
+VisibleForRuntimeDeployment: no
+
+Architecture: windows_x64
+Built-Using: nipkg
+CompatibilityVersion: 240500
+Description: mycompany-myproduct
+DisplayName: SourceMeasureDCVoltage
+Filename: https://github.com/NI-Measurement-Plug-Ins/abstraction-layer-labview/releases/download/v1.0.0.1/ni-source-measure-dcvoltage_1.0.0.1_windows_x64.nipkg
+MD5sum: a56187d21b56d486fce48d9dbdf9d9fc
+Maintainer: Github Community <santhosh.ramaraj@ni.com>
+Package: ni-source-measure-dcvoltage
+Plugin: file
+Section: Add-Ons
+Size: 20213770
+StoreProduct: no
+UserVisible: yes
+Version: 1.0.0.1
+VisibleForRuntimeDeployment: no
+
```

Comment:
@jayaseelan-james , need to fix the Description display name, maintainer etc. Python packages are good.

Reply:
Sure. I've raised a separate PR #31 only containing the Python measurement packages.